### PR TITLE
gcp - fix managed zones get parameters

### PR DESCRIPTION
**Pull request by alexkarpitski**
- Add `query` and `get` methods for `dns project` resource.
- Rename existing test flight data to comply with the agreed conventions.
- Use `zone_name` instead of `name` for getting `managed zone` resources.

### CODE REVIEW

File: tools/c7n_gcp/c7n_gcp/resources/dns.py [hunk 1]
```diff
@@ -49,3 +51,23 @@ def get(client, resource_info):
             return client.execute_query(
                 'get', {'project': resource_info['project_id'],
                         'policy': resource_info['name']})
+
+
+@resources.register('dns-project')
+class DNSProject(QueryResourceManager):
```
Comment (kapilt) [inline tools/c7n_gcp/c7n_gcp/resources/dns.py]: Whats the use case for this resource, its read only...

Comment (alexkarpitski) [reply tools/c7n_gcp/c7n_gcp/resources/dns.py]: @jonathana, @jondavidi - your assistance with a use case will be appreciated.

Comment (alexkarpitski) [reply tools/c7n_gcp/c7n_gcp/resources/dns.py]: @kapilt, agreed to remove the resource, thank you.

**Comment by kapilt** [comment]
i think this also needs an api event for test data to ensure the get method is high fidelity to the actual api call.

**Comment by alexkarpitski** [comment]
> 
> 
> i think this also needs an api event for test data to ensure the get method is high fidelity to the actual api call.
Sure, provided the event data.

**Pull request merged**